goal_src/engine/game/settings.gc

```diff
@@ -5,5 +5,734 @@
 ;; name in dgo: settings
 ;; dgos: GAME, ENGINE
 
-;; TODO - for video
-(define-extern *setting-control* setting-control)
+;; definition for method 9 of type setting-data
+(defmethod TODO-RENAME-9 setting-data ((obj setting-data) (arg0 engine))
```

Comment:
Possible name: `handle-pending-updates`: this goes through the list of pending setting changes in the `engine`/connection list and updates the `setting-data`.